src/test/groovy/com/getbase/services/BaseSpecification.groovy

```diff
@@ -86,6 +86,8 @@ abstract class BaseSpecification extends Specification {
                 'value'     : 1000.0,
                 'tags'      : ["important"],
                 'contact_id': createContact().id,
+                'last_stage_change_at' : "2014-09-27T16:32:56+00:00",
```

Comment:
Please keep the format of date consistent w/ the official documentation (all 4 dates).